src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -27,6 +27,14 @@ class VarDumper
 
     public static function dump($var)
     {
+        $args = func_get_args();
+        if (count($args) > 1) {
+            foreach ($args as $arg) {
+                self::dump($arg);
+            }
+            return false;
```

Comment:
this introduces a mixed api, should probably return `void` or an array of all results.